### PR TITLE
Allow newer version of signxml

### PR DESCRIPTION
**Pull request by jjjacksn**
In the effort to update our xml related libraries we need to permit a newer version of signxml within this library.

### CODE REVIEW

**Comment by jjjacksn** [comment]
* **#2** <a href="https://app.graphite.dev/github/pr/dave-energy/openleadr-python/2?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @jjjacksn and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: setup.py [hunk 1]
```diff
@@ -16,19 +16,31 @@
...
+    },
+    packages=["openleadr", "openleadr.service"],
+    python_requires=">=3.7.0",
+    include_package_data=True,
+    install_requires=[
+        "xmltodict==0.13.0",
+        "aiohttp>=3.8.3,<4.0.0",
+        "apscheduler>=3.10.0,<4.0.0",
+        "jinja2>=3.1.2,<4.0.0",
+        "signxml>=3.1.0,<3.3.0",
```
Comment (vel9) [inline setup.py:41]: The change seems good. Just curious re: how we're going to test? Any tests events we can play?

**Review by vel9 — APPROVED** [review]
Approving - just left a question re: testing

File: setup.py (hunk 1 above)
Comment (jjjacksn) [reply setup.py:41]: 🤔  not sure

Comment (jjjacksn) [reply setup.py:41]: @tedsmoore any thoughts in how we might test changes to the OpenADR implementation? I see that we are still getting regular pings to the `/api/OpenADR2/Simple/EiEvent` endpoint.

**Pull request merged**